### DISCUSSION TITLE
favoriot : Issue displaying data from favoriot IoT platform

### DISCUSSION THREAD

**Discussion by Coderakid01** [grafana/grafana-infinity-datasource#145] [Specific APIs]
Greetings, I had a problem in linking IoT Platform FAVORIOT to Grafana by using this Plugin. It appear the plugin unable to display the data on my Grafana Account but when i tried on the example Plugin from your website the data is available. Is there any sort of solution for this part???

**Comment by yesoreyeram** [comment]
Have you tried the same way as below? 
 
Here is the config:
![image](https://user-images.githubusercontent.com/153843/132570983-3ffd224b-228e-455f-b107-413ed78031c5.png)

Here is the query:
![image](https://user-images.githubusercontent.com/153843/132570941-acbe1d84-00de-4c74-bf00-10786ef703b0.png)

**Comment by Coderakid01** [comment]
Hi Thanks for the response.
Unfortunately It didn't work on my side... I would love to see a video demo
for better references.
